packages/core/src/Carousel/Carousel.styles.ts

```diff
@@ -133,7 +133,7 @@ export const { staticClasses, useClasses } = createClasses("HvCarousel", {
     width: 5,
     height: 5,
     margin: theme.space.xs,
-    borderRadius: "50%",
+    borderRadius: theme.radii.circle,
```

Comment:
We should probably deprecate `circle` in favor of `full`. In most cases where this is used, we'll have a `width=height`, so they'll be the same.

```suggestion
    borderRadius: theme.radii.full,
```

In cases where `width!=height`, we probably don't want `circle` anyways, because it'll lead to weird UI. Example of `circle` vs `full` in HvButton:
![image](https://github.com/user-attachments/assets/134572e9-49cb-4469-af8e-2159b44efbd8) ![image](https://github.com/user-attachments/assets/6def4f26-e843-45a9-9149-199570d671f7)

Tailwind also has a `none` and a `full`, but (from what I can see) no utility for "circle":
https://tailwindcss.com/docs/border-radius

Reply:
Yeah, I agree. I see no use case for the `circle` variant when width and height are not the same. Also, we could deprecate `none`? I see no semantic value in it.